src/templates/bs5/components/alert/default.html

```diff
@@ -1,3 +1,3 @@
-<div class="alert alert-${1:primary|secondary|success|danger|warning|info|light|dark}" role="alert">
+<div class="alert ${1|alert-primary,alert-secondary,alert-success,alert-danger,alert-warning,alert-info,alert-light,alert-dark|}" role="alert">
```

Comment:
Do we need really this?

Can we keep this:
```html
<div class="alert alert-${1:primary|secondary|success|danger|warning|info|light|dark}" role="alert">
```

Instead of this:
```html
<div class="alert ${1|alert-primary,alert-secondary,alert-success,alert-danger,alert-warning,alert-info,alert-light,alert-dark|}" role="alert">
```

This will trigger like this: 

```html
<div class="alert alert-CURSOR-HERE" role="alert"></div>
```

why do we need to type `alert-` unnecessarily?

Reply:
Yes we need to Do that otherwise suggestions come empty. I guess after - our suggestions don't work. You can try on vs code.

Reply:
Yes, with your method, the suggestions are displayed. Thank you.